src/screens/Member/signin.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {
+    View,
+    Text,
+    TextInput, TouchableOpacity, KeyboardAvoidingView, Platform
+} from "react-native";
+import styles from "./styles";
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {NavigationScreenProp} from "react-navigation";
+
+
+interface IProps {
+    navigation: NavigationScreenProp<any, any>
+}
+
+export default class Signin extends React.PureComponent<IProps, any> {
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            emailOrPhone: "",
+            password: "",
+            showPassword: false,
+            validate: null
+        }
+    }
+
+    // @ts-ignore
+    componentDidUpdate() {
+        this.validate();
+    }
+
+    validate = () => {
+        const {emailOrPhone, password} = this.state;
+
+        if (emailOrPhone.length > 5 && password.length > 5) {
+            this.setState({
+                validate: true
+            })
+        } else {
+            this.setState({
+                validate: false
+            })
+        }
```

Comment:
`if()...else{...}` can be simplified with  a single line expression. 
`this.setState({validate: emailOrPhone.length > 5 && password.length > 5})